src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandContext.java

```diff
@@ -281,6 +289,7 @@ public class Builder<SchemaT extends SchemaObject> {
       private String commandName;
       private RequestContext requestContext;
       private ApiFeatures apiFeatures;
+      private MeterRegistry meterRegistry;
```

Comment:
I think this should be on the BuilderSupplier - it's the same for every request.

and it must be non null